_data/team/roles/continuous_integration.yml

```diff
@@ -0,0 +1,13 @@
+role: Continuous integration
+tasks:
+  - Building and developing new CI infrastructure
+  - Monitoring CI status
+  - Maintenance and fixes
+current_leads:
+  - Irfan Alibay
+  - Richard Gowers
+current_members:
+  - Fiona Naughton
+historical_members:
+  - Irfan Alibay
+
```

Comment:
I think I might have even set up the first MDA CI ever...

Reply:
Resolving as we still need to settle how we want to acknowledge historical contributors for now.